Source/WebKit/GPUProcess/media/RemoteMediaPlayerManagerProxy.cpp

```diff
@@ -217,9 +217,10 @@ ShareableBitmapHandle RemoteMediaPlayerManagerProxy::bitmapImageForCurrentTime(W
 
     context->drawNativeImage(*image, imageSize, FloatRect { { }, imageSize }, FloatRect { { }, imageSize });
 
-    ShareableBitmapHandle bitmapHandle;
-    bitmap->createHandle(bitmapHandle);
-    return bitmapHandle;
+    auto bitmapHandle = bitmap->createHandle();
```

Comment:
This method should return a std::optional<ShareableBitmap> to be consistent.
Would also help never having to deal With invalid handle (and we should remove this possibility)

Reply:
I meant to put this comment above, but github won't let me